geopandas/sindex.py

```diff
@@ -70,6 +89,129 @@ def __init__(self, geometry):
                 # See https://github.com/Toblerity/rtree/issues/20.
                 super().__init__()
 
+            # store reference to geometries for predicate queries
+            self._geometries = geometry.geometry.values
+            # create a prepared geometry cache
+            self._prepared_geometries = np.array([None] * self._geometries.size)
+
+        def query(self, geometry, predicate=None, sort=False):
+            """Compatibility layer for pygeos.query.
+            This is not an optimized function, if speed is important,
+            you are probably better off using PyGEOS.
+            Parameters
+            ----------
+            geometry : shapely geometry
+                A single shapely geometry to query against the spatial index.
+            predicate : {None, 'intersects', 'within', 'contains', 'overlaps', 'crosses', 'touches'}, optional
```

Comment:
```suggestion
            predicate : {None, 'intersects', 'within', 'contains', 'overlaps', \
'crosses', 'touches'}, optional
```

That's a (somewhat ugly) way to solve the "too long line length" (instead of adding `# noqa: E501`)

Reply:
I think that came from `black` or something, I _have_ `#noqa: E501` there. But I'll fix it.

Reply:
Okay, no problem. I switched to `\` and fixed all of the other long lines.